app/models/admin_blacklisted_email.rb

```diff
@@ -0,0 +1,36 @@
+class AdminBlacklistedEmail < ActiveRecord::Base
+  attr_accessible :email
+
+  before_validation :canonicalize_email
+  
+  validates :email, presence: true, uniqueness: true, email_veracity: true
+
+  def canonicalize_email
+    self.email = AdminBlacklistedEmail.canonical_email(self.email) if self.email
+  end
+  
+  # Produces a canonical version of a given email reduced to its simplest form
+  # This is what we store in the db, so that if we subsequently check a submitted email,
+  #  we only need to clean the submitted email the same way and look for it.
+  def self.canonical_email(email_to_clean)
+    canonical_email = email_to_clean.downcase
+    canonical_email.strip!
+    canonical_email.sub!('@googlemail.','@gmail.')
+
+    # strip periods from gmail addresses
+    if (matchdata = canonical_email.match(/(.+)\@gmail\.com/))
+      canonical_email = matchdata[1].sub('.', '') + "@gmail.com"
```

Comment:
I do have tests (in the spec) but missed this variant, whoops. Any other suggestions to test?